packages/amplication-data-service-generator/README.md

```diff
@@ -20,23 +20,58 @@ The generator can be used as a library or as a CLI.
 - [Recast](https://github.com/benjamn/recast): is used for generating the TypeScript code.
 - [Jest](https://jestjs.io/): is used for testing the generator code and for testing in the generated code.
 
-## Development
+### Development
 
-### Testing
+:bulb: Before you begin, make sure you have all the below installed:
 
-#### Run unit and snapshot tests
+- [Node.js v14 or above](https://nodejs.org/en/download/)
+- [npm v7 or above](https://github.blog/2020-10-13-presenting-v7-0-0-of-the-npm-cli/)
+- [Docker](https://docs.docker.com/desktop/)
+- [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git/)
 
-To run the unit and snapshot tests of the generator execute:
+#### One time set up
 
-```
-npm test
-```
+- Install dependencies of the monorepo (execute in root directory):
+  ```
+  npm install
+  npm run bootstrap
+  ```
+- Build the Data Service Generator and its dependencies:
+  ```
+  npm run build -- --scope @amplication/data-service-generator --include-dependencies
+  ```
+- Open the data service generator directory
+  ```
+  cd packages/amplication-data-service-generator
+  ```
+- Generate the test data service app
+  ```
+  npm run generate-test-data-service
+  ```
 
-To view more options for running tests execute:
+#### Workflow
 
-```
-npm test -- --help
-```
+Make sure you are in the data service generator directory (`packages/amplication-data-service-generator`).
```

Comment:
Maybe specify the root directory in line 34 instead?

Reply:
I've added a sentence in line 34.